delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/remote/client/RemoteDiskStorageClient.scala

```diff
@@ -35,39 +37,20 @@ import scala.concurrent.duration._
 /**
   * The client to communicate with the remote storage service
   */
-final class RemoteDiskStorageClient(client: HttpClient, getAuthToken: AuthTokenProvider, credentials: Credentials)(
-    implicit as: ActorSystem
-) {
-  import as.dispatcher
-
-  private val serviceName = Name.unsafe("remoteStorage")
+trait RemoteDiskStorageClient {
```

Comment:
This is being a dependency of storages and files, that seems the best compromise to do the refactoring without spending too much time on it.